docs/dev/ci/github_actions/security.md

```diff
@@ -0,0 +1,24 @@
+# Security best practices for GitHub Actions Workflows
+
+There are a few simple steps that we should follow to ensure our workflows are not vulnerable to common attacks:
+
+- Use the “permissions” key to make sure the GITHUB_TOKEN is configured with the least privileges for each job. Check [GitHub documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token) on this also.
```

Comment:
```suggestion
- Use the `permissions` key to make sure the `GITHUB_TOKEN` is configured with the least privileges for each job. Check [GitHub documentation](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token) on this also.
```
I would also suggest adding the example from either our workflows or from the GH docs.

Reply:
Added